docs/release/dev/sp-fix-utf-text.md

```diff
@@ -0,0 +1,3 @@
+Fixed displaying of non-ASCII text: fixed working with UTF-8 (from JS) text and recompiled
+font to include Russian symbols. In case you wish to update the font to use something else,
+please see [SP Texts API docs](../texts.md#compiling-font).
```

Comment:
This should become clickable once these lines will move from `docs/release/dev/` to `docs/release/`